### DISCUSSION TITLE
Can CLI arguments be added in wingetcreate?

### DISCUSSION THREAD

**Discussion by Josh65-2201** [microsoft/winget-create#50] [Q&A]
I want to add the oculus software to the list of programs but I need to add /unattended to the exe so it does silent install and is compatible but I don't know how to specify that in the manifest. Is it even possible?

**Comment by denelon** [comment]
@josh-65 I moved this to the winget-create repository. The tool was designed to detect the installer type so switches aren't necessary. In the case of a .exe installer that isn't recognized, we may need to have this as part of the "optional fields" feature.

**Reply by Josh65-2201** [reply]
it works mostly however oculus software only has an exe and doesn't work. Doing it manual with a /unattended switch makes it work fine. I also tried Marcum reflect witch need some switch's to define settings for silent install.

I made an issue for this
https://github.com/microsoft/winget-create/issues/51#issue-908418900
